snap/info.go

```diff
@@ -19,24 +19,31 @@
 
 package snap
 
+import (
+	"github.com/ubuntu-core/snappy/timeout"
+)
+
 // Info provides information about snaps.
 type Info struct {
-	Name            string
-	Developer       string
-	Version         string
-	Revision        int
-	Type            Type
-	Channel         string
-	Description     string
-	Summary         string
-	Apps            map[string]*AppInfo
-	Plugs           map[string]*PlugInfo
-	Slots           map[string]*SlotInfo
-	Sha512          string
-	Size            int64
-	AnonDownloadURL string
-	DownloadURL     string
-	IconURL         string
+	Name             string
+	Developer        string
+	Version          string
+	Revision         int
+	Type             Type
+	Architectures    []string
+	Channel          string
+	Description      string
+	Summary          string
+	LicenseAgreement string
+	LicenseVersion   string
+	Apps             map[string]*AppInfo
+	Plugs            map[string]*PlugInfo
+	Slots            map[string]*SlotInfo
```

Comment:
Can we please insert an empty line and then this comment before the following block:
```
// The information in these fields is not present inside the snap blob itself.
```
Then, let's please move Revision, Developer, and Channel to the top of this block.